src/app/static/src/js/components/permissions/roleList.vue

```diff
@@ -15,8 +15,7 @@
                        :users="role.members"
                        :canRemove="canRemoveMembers"
                        @removed="function(email){removeMember(role.name, email)}"></user-list>
-            
-            <error-info :default-message="defaultMessage" :api-error="error"></error-info>
+
```

Comment:
..and actually multiple error messages had crept back in here, probably as the result of a merge